src/plugins/validate-semantic/validators/2and3/parameters.js

```diff
@@ -73,3 +73,48 @@ export const validate2And3ParameterDefaultsMatchAnEnum = () => (system) => {
       }, [])
     })
 }
+
+export const validate2And3PathParameterIsDefinedInPath = () => (system) => {
+  const refArray = []
+  return system.validateSelectors
+    .allParameters()
+    .then(nodes => {
+      return nodes.reduce((acc, node) => {
+        const parameter = node.node || {}
+        const path = node.path
+        const isFromPath = path[0] === "paths" ? true : false
+        const pathString = path[1]
+        const paramName = parameter.name
+        const paramInPath = "{" +  paramName + "}"
+        const ref = parameter.$ref
+        if (parameter !== undefined) {
```

Comment:
this `undefined` check seems extraneous, since you assign an initial value in line 83 above

Reply:
Done! Thanks!